### ISSUE TITLE
Issues creating a lb_monitor_v2 using current documentation

### ISSUE THREAD

**Issue by Moep90** [opentelekomcloud/terraform-provider-opentelekomcloud#424]
I try to create a lb_monitor_v2 according to the documentation but it fails.

### Terraform Version
```
/usr/local/bin/terraform --version
Terraform v0.11.14
+ provider.dns v2.2.0
+ provider.local v1.4.0
+ provider.null v2.1.2
+ provider.openstack v1.23.0
+ provider.opentelekomcloud v1.14.0
+ provider.template v2.1.2

```
### Affected Resource(s)
- opentelekomcloud_lb_monitor_v2

### Terraform Configuration Files

main.tf

```hcl
resource "opentelekomcloud_lb_monitor_v2" "monitor" {
  pool_id        = "${var.pool_id}"
  name           = "${var.name}"
  tenant_id      = "${var.tenant_id}"
  type           = "${var.type}"
  delay          = "${var.delay}"
  timeout        = "${var.timeout}"
  max_retries    = "${var.max_retries}"
  url_path       = "${var.url_path}"
  http_method    = "${var.http_method}"
  expected_codes = "${var.expected_codes}"
  admin_state_up = "true"
}
```
module call:
```hcl
module "elb_webapps_monitor" {
  source = "terraform/modules/terraform-module-elb_enhanced_monitor"

  pool_id        = "${module.elb_webapps_pool.id}"
  name           = "${var.project_name}-${var.stage}-elb-monitor"
  type           = "HTTP"
  delay          = "7"
  timeout        = "5"
  max_retries    = "5"
  url_path       = "/"
  http_method    = "GET"
  expected_codes = "200-202"
}
```
#### Documentation
```
expected_codes - (Optional) Required for HTTP(S) types. Expected HTTP codes for a passing HTTP(S) monitor. You can either specify a single status like "200", or a range like "200-202".
```
### Debug Output
```
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: 2019/12/10 12:33:01 [DEBUG] OpenTelekomCloud Request Body: {
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:   "healthmonitor": {
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "admin_state_up": true,
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "delay": 7,
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "expected_codes": "200-202",
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "http_method": "GET",
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "max_retries": 5,
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "name": "<PROJECT>-dev-elb-monitor",
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "pool_id": "92c7b24d-c277-43dc-ade4-4b5b527df438",
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "timeout": 5,
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "type": "HTTP",
2019-12-10T12:33:01.143+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "url_path": "/"
2019-12-10T12:33:01.144+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:   }
2019-12-10T12:33:01.144+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: }
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: 2019/12/10 12:33:01 [DEBUG] OpenTelekomCloud Response Code: 400
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: 2019/12/10 12:33:01 [DEBUG] OpenTelekomCloud Response Headers:
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: Content-Length: 146
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: Content-Type: application/json
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: Date: Tue, 10 Dec 2019 11:33:01 GMT
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: Server: Web Server
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: X-Openstack-Request-Id: req-5d20f738-6f02-43cb-8d20-59f0249b5d20
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: 2019/12/10 12:33:01 [DEBUG] OpenTelekomCloud Response Body: {
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:   "NeutronError": {
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "detail": "",
2019-12-10T12:33:01.481+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "message": "Invalid input for expected_codes. Reason: '200-202' is not a valid input.",
2019-12-10T12:33:01.482+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:     "type": "HTTPBadRequest"
2019-12-10T12:33:01.482+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4:   }
2019-12-10T12:33:01.482+0100 [DEBUG] plugin.terraform-provider-opentelekomcloud_v1.14.0_x4: }
```
### Panic Output
```
Error: Error applying plan:
2019/12/10 12:33:07 [DEBUG] plugin: waiting for all plugin processes to complete...

2019-12-10T12:33:07.435+0100 [WARN ] plugin: error closing client during Kill: err="connection is shut down"
2019-12-10T12:33:07.436+0100 [DEBUG] plugin.terraform-provider-openstack_v1.23.0_x4: 2019/12/10 12:33:07 [ERR] plugin: plugin server: accept unix /tmp/plugin093936135: use of closed network connection
2019-12-10T12:33:07.440+0100 [DEBUG] plugin: plugin process exited: path=<HOME>/<PATH_TO_REPO>/.terraform/plugins/linux_amd64/terraform-provider-openstack_v1.23.0_x4
1 error occurred:
	* module.elb_webapps_monitor.opentelekomcloud_lb_monitor_v2.monitor: 1 error occurred:
	* opentelekomcloud_lb_monitor_v2.monitor: unexpected EOF

```
### Steps to Reproduce

1. `terraform apply`

**Comment by ShiChangkuo** [comment]
hello @Moep90 ,  thanks a lot for raising, the 'expected_codes' parameter is reserved and has been not used according to the [api websit](https://docs.otc.t-systems.com/en-us/api/elb/en-us_topic_0096561563.html), and I will update our document soon.

**Comment by niuzhenguo** [comment]
closed by #430
